solrman/smservice/run.go

```diff
@@ -239,6 +239,11 @@ func (s *SolrManService) setStatusOp(status string) {
 	}
 }
 
+// For admin visibility
+func (s *SolrManService) GetStatusOp() *solrmanapi.OpRecord {
+	return s.statusOp
```

Comment:
For thread-safety, you must hold the lock to safely read this. Also, since this is exported, we probably want to make a defensive copy of the op.